.github/workflows/build-tests.yml

```diff
@@ -43,5 +43,46 @@ jobs:
       run: |
         dnf localinstall -y /tmp/RPMS/*
 
-    - name: Build IDM Console Framework
+    - name: Build with Ant
       run: ./build.sh
+
+    - name: Install JSS into local Maven repo
+      run: |
+        # get JSS <major>.<minor>.<update> version
+        rpm -q dogtag-jss | tee output
+        JSS_VERSION=$(sed -e 's/^dogtag-jss-\([0-9]\+\.[0-9]\+\.[0-9]\+\).*$/\1/' output)
+
+        # if built by COPR, jss-base.jar will be installed in /usr/lib/java,
+        # otherwise it will be in /usr/share/java.
+        JSS_BASE_JAR=$(find /usr/lib/java /usr/share/java -name jss-base.jar)
+
+        mvn install:install-file \
+            -Dfile=$JSS_BASE_JAR \
+            -DgroupId=org.dogtagpki.jss \
+            -DartifactId=jss-base \
+            -Dversion=$JSS_VERSION-SNAPSHOT \
+            -Dpackaging=jar \
+            -DgeneratePom=true
+
+    - name: Install LDAP JDK into local Maven repo
+      run: |
+        # get LDAP JDK <major>.<minor>.<update> version
+        rpm -q dogtag-ldapjdk | tee output
+        LDAPJDK_VERSION=$(sed -e 's/^dogtag-ldapjdk-\([0-9]\+\.[0-9]\+\.[0-9]\+\).*$/\1/' output)
```

Comment:
This can be simplified to `LDAPJDK_VERSION=$(rpm -q --qf "%{Version}" dogtag-ldapjdk)`